report/common.py

```diff
@@ -358,6 +364,28 @@ def get_targets(self, benchmark: str, sample: str) -> list[Target]:
 
     return targets
 
+  def _get_targets_agent(self, benchmark: str, trial: str) -> Target:
+    """Gets the targets of benchmark |benchmark| with trial ID |trial| from
+    the OFG version 2 (LLM agents)."""
+    fuzz_target_dir = os.path.join(self._results_dir, benchmark, 'fuzz_target')
+    files = sorted(FileSystem(fuzz_target_dir).listdir())
+
+    fuzz_target_code = ''
+    if f'{trial:02s}.fuzz_target' in files:
+      fuzz_target_path = os.path.join(fuzz_target_dir,
+                                      f'{trial:02s}.fuzz_target')
+      with FileSystem(fuzz_target_path).open() as f:
+        fuzz_target_code = f.read()
+
+    build_script_code = ''
+    if f'{trial:02s}.build_script' in files:
+      build_script_path = os.path.join(fuzz_target_dir,
+                                       f'{trial:02s}.build_script')
+      with FileSystem(build_script_path).open() as f:
+        build_script_code = f.read()
+
+    return Target(code=fuzz_target_code, fixer_prompt=build_script_code)
```

Comment:
why `fixer_prompt` ?

Reply:
It's a temporary hacky way to show the new build script in the report.
I plan to organize it, but maybe better in a separate PR?

Reply:
Is it a lot of work to do that ? 

If so, please add TODO with an explanation here.